packages/frontend-shared/cypress/e2e/support/e2eProjectDirs.ts

```diff
@@ -26,8 +26,10 @@ export const e2eProjectDirs = [
   'ids',
   'integration-outside-project-root',
   'issue-8111-iframe-input',
+  'kill-child-process',
   'launchpad',
   'max-listeners',
+  'migration',
   'multiple-task-registrations',
```

Comment:
We need to have this as a Git Hook because it is starting to get out of control...